packages/ove-app-charts/README.md

```diff
@@ -0,0 +1,55 @@
+# Charts App
+
+This app supports visualisation of charts (excluding network diagrams) using the OVE framework. It is based on [Vega](https://vega.github.io/vega/), a declarative language for creating, saving, and sharing interactive visualization designs. Vega supports Canvas and SVG rendering.
+
+## Application State
+
+The state of this app has a format similar to what is provided below:
+
+```json
+{
+    "spec": "https://raw.githubusercontent.com/vega/vega/master/docs/examples/bar-chart.vg.json",
+    "options": {
+        "width": 800,
+        "height": 800
+    }
+}
+```
+
+The `spec` field points to either a URL of, or contains an embedded [Vega specification](https://vega.github.io/vega/docs/#specification).
+
+## Loading the App
+
+A chart can be loaded using the OVE APIs:
+
+Windows:
```

Comment:
* If the Windows and Linux/Commands are identical, they should not be duplicated: display them once, with a title like "Windows or Unix-like (Linux/Mac)"

* Why have you written the ``data`` for each request within double quotes (and thus escaped all the quotes inside), rather than with single quotes? i.e., why  `"{\"h\": 500 ... "` rather than `'{"h": 500'` ?

The same comments apply to the other READMEs.

**Edit:** I've seen that the reason is that you only use the more verbose quoting for Windows. I'm surprised using single quotes command doesn't work for Windows as it does for Linux/Mac (@davidbirchwork).

Reply:
@jamesscottbrown, running curl on the Windows CMD requires escaping quotes as explained. This is also discussed in several SO posts. You can try this for yourself to be certain.